src/plots/cartesian/axes.js

```diff
@@ -1799,12 +1811,18 @@ function numFormat(v, ax, fmtoverride, hover) {
         // 0 is just 0, but may get exponent if it's the last tick
         v = '0';
         isNeg = false;
-    }
-    else {
+    } else {
```

Comment:
Ha.

I still like 

```js
if() {

}
else {

}
```

better. But, right,  I'm sure @bpostlethwaite would agree that your way is better.